kf/bpf.go

```diff
@@ -180,18 +180,18 @@ func LoadRootProgram(ifaceName string, direction string, progType string, conf *
 	if fileExists(rootProgBPF.MapNamePath) {
 		log.Warn().Msgf("previous instance of root program %s persisted map %s file exists", rootProgBPF.Program.Name, rootProgBPF.MapNamePath)
 		if err := rootProgBPF.RemoveRootProgMapFile(ifaceName); err != nil {
-			log.Info().Msgf("previous instance of root program %s map file removed successfully - %s ", rootProgBPF.Program.Name, rootProgBPF.MapNamePath)
+			log.Warn().Msgf("previous instance of root program %s map file removed unsuccessfully - %s ", rootProgBPF.Program.Name, rootProgBPF.MapNamePath)
```

Comment:
We are not exposing the original error here. I see below we are using the following method:
```go
log.Warn().Err(err).Msgf("user mode BPF program failed - %s", b.Program.Name)
```
Where we are embedding the error using the `log.Err()` function. Should we do that here?

Reply:
Done